cluster/log-dump/log-dump.sh

```diff
@@ -323,8 +324,8 @@ function dump_nodes() {
     node_names=( "$@" )
   elif [[ -n "${use_custom_instance_list}" ]]; then
     echo "Dumping logs for nodes provided by log_dump_custom_get_instances() function"
-    node_names=( $(log_dump_custom_get_instances node) )
-  elif [[ ! "${node_ssh_supported_providers}" =~ "${KUBERNETES_PROVIDER}" ]]; then
+    while IFS='' read -r line; do node_names+=("$line"); done < <(log_dump_custom_get_instances node)
```

Comment:
```
In ./cluster/log-dump/log-dump.sh line 326:
    node_names=( $(log_dump_custom_get_instances node) )
                 ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```